### PR TITLE
Added asset status field to device-info form.

### PR DESCRIPTION
**Pull request by ghost**
The field show read only status (if asset is linked to displayed device) or
empty string.

connected to: #1666

### CODE REVIEW

File: src/ralph/ui/forms/devices.py [hunk 1]
```diff
@@ -397,6 +404,16 @@ class Meta(DeviceForm.Meta):
             'deleted',
         )
 
+    def set_asset_status(self):
+        asset_status = ''
+        linked_asset = self.instance.get_asset()
+        if linked_asset:
+            asset_status = (
+            AssetStatus.NameFromID(linked_asset.status) if
```
Comment (mkurek) [inline src/ralph/ui/forms/devices.py]: fix indentation here (weird - flake8 should handle it :expressionless:)

Comment (mkurek) [inline src/ralph/ui/forms/devices.py]: afaik there is underscore version of this method (`name_from_id`), which is more pythonic - could you use it?

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**